trl/models/modelling_sd_base.py

```diff
@@ -1,5 +1,19 @@
+# Copyright 2023 DDPO-pytorch authors (Kevin Black), The HuggingFace Team, metric-space. All rights reserved.
```

Comment:
@younesbelkada just a note here: I changed the order of authors names as the default implementation code  is mostly from the diffusers library